test/object-store/util/sync/baas_admin_api.cpp

```diff
@@ -268,6 +268,32 @@ size_t curl_header_cb(char* buffer, size_t size, size_t nitems, std::map<std::st
     return nitems * size;
 }
 
+std::string_view getenv_sv(const char* name) noexcept
+{
+    if (auto ptr = ::getenv(name); ptr != nullptr) {
+        return std::string_view(ptr);
+    }
+
+    return {};
+}
+
+static std::string unquote_string(std::string_view possibly_quoted_string)
```

Comment:
Can we make this a public shared utility function from sync_test_utils.hpp instead of copying it?

Reply:
Done.